src/BuildScriptGenerator/DotNetCore/DotNetCorePlatformInstaller.cs

```diff
@@ -47,8 +47,9 @@ public override void InstallPlatformSpecificSkeletonDependencies(StringBuilder s
                 "zlib1g",
                 "libuuid1",
                 "libunwind8");
-
+            stringBuilder.AppendLine("if grep -q cli \"/opt/oryx/.imagetype\"; then");
```

Comment:
This change means we no longer install Python for the Jamstack image, correct?

Reply:
Yep, correct. We would only install SDKs for versions other than 3.8